### PR TITLE
Update teams-powershell-overview.md

### PR DESCRIPTION
**Pull request by pupara**
Updated Content

### CODE REVIEW

**Comment by Kateyanne** [comment]
@JohanFreelancer9 could you please copyedit? @scanum fyi

**Review by get-itips — APPROVED** [review]

File: Teams/teams-powershell-overview.md [hunk 1]
```diff
@@ -17,38 +17,24 @@ appliesto:
 
 # Microsoft Teams PowerShell Overview
 
-Microsoft Teams PowerShell is a set of cmdlets for managing Teams directly from the PowerShell command line. Written in .NET Standard, Teams PowerShell works on PowerShell 5.1 on Windows, PowerShell 6.x and higher on all platforms including Azure Cloud Shell.
+Microsoft Teams PowerShell module is a set of cmdlets for managing Teams directly from the PowerShell command line. PowerShell provides powerful features for automation that can be leveraged for managing your Teams workload.  
 
-Before you can start using PowerShell, you'll need to [install it](teams-powershell-install.md). 
+You can use the Microsoft Teams PowerShell module with one of the following methods:
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-overview.md:22]:
```suggestion
You can use the Microsoft Teams PowerShell module using one of the following methods: 
```

File: Teams/teams-powershell-overview.md [hunk 2]
```diff
@@ -17,38 +17,24 @@ appliesto:
...
-Microsoft Teams PowerShell is a set of cmdlets for managing Teams directly from the PowerShell command line. Written in .NET Standard, Teams PowerShell works on PowerShell 5.1 on Windows, PowerShell 6.x and higher on all platforms including Azure Cloud Shell.
+Microsoft Teams PowerShell module is a set of cmdlets for managing Teams directly from the PowerShell command line. PowerShell provides powerful features for automation that can be leveraged for managing your Teams workload.  
 
-Before you can start using PowerShell, you'll need to [install it](teams-powershell-install.md). 
+You can use the Microsoft Teams PowerShell module with one of the following methods: 
 
-> [!WARNING]
-> There are known issues with PowerShell 7 and Teams PowerShell. We recommend using PowerShell 5.1 until the issues are resolved.
+- Install the Microsoft Teams PowerShell module via [PowerShell Gallery](https://www.powershellgallery.com/packages/MicrosoftTeams) (recommended option). 
+- Install the Microsoft Teams PowerShell module with [Manual Download](https://www.powershellgallery.com/packages/MicrosoftTeams).
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-overview.md:25]:
```suggestion
- Install the Microsoft Teams PowerShell module via [Manual Download](https://www.powershellgallery.com/packages/MicrosoftTeams). 
```

File: Teams/teams-powershell-overview.md [hunk 3]
```diff
@@ -17,38 +17,24 @@ appliesto:
...
 
-## Releases
 
+# Features 
 
-Teams PowerShell is available on [PowerShell Gallery](https://www.powershellgallery.com/packages/MicrosoftTeams) in two release types.
+The Microsoft Teams PowerShell module includes the following features: 
 
-- **General Availability (GA)**: Production-ready cmdlets, updated monthly.
+- Uses a single module to manage all aspects of Teams administration including users, teams, policies and configuration.
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-overview.md:32]:
```suggestion
- Uses a single module to manage all aspects of Teams administration, including users, teams, policies, and configuration.  
```

File: Teams/teams-powershell-overview.md [hunk 4]
```diff
@@ -17,38 +17,24 @@ appliesto:
...
+You can use the Microsoft Teams PowerShell module with one of the following methods: 
 
-> [!WARNING]
-> There are known issues with PowerShell 7 and Teams PowerShell. We recommend using PowerShell 5.1 until the issues are resolved.
+- Install the Microsoft Teams PowerShell module via [PowerShell Gallery](https://www.powershellgallery.com/packages/MicrosoftTeams) (recommended option). 
+- Install the Microsoft Teams PowerShell module with [Manual Download](https://www.powershellgallery.com/packages/MicrosoftTeams). 
 
-## Releases
 
+# Features
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-overview.md:28]:
```suggestion
## Features 
```

File: Teams/teams-powershell-overview.md [hunk 5]
```diff
@@ -17,38 +17,24 @@ appliesto:
...
-
-- [Microsoft Teams PowerShell module](https://www.powershellgallery.com/packages/MicrosoftTeams/): The Teams PowerShell module contains cmdlets for managing teams, chat, and channels.
-
-> [!NOTE]
-> The [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/) version 2.0 or higher includes all  Skype for Business Online Connector cmdlets, providing a single module for Teams PowerShell management.
-
-- [Skype for Business PowerShell Connector](/microsoft-365/enterprise/manage-skype-for-business-online-with-microsoft-365-powershell): The Skype for Business PowerShell connector is now a part of Teams PowerShell module.
-
-For a complete guide to managing Teams using these modules, please see [Manage Teams with Teams PowerShell](teams-powershell-managing-teams.md).
+# Data collection
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-overview.md:35]:
```suggestion
##  Data collection 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pupara, please commit my suggested changes. Many thanks.

**Pull request merged**